solidity/ecdsa/deploy/03_deploy_wallet_registry.ts

```diff
@@ -74,7 +74,7 @@ export default func
 func.tags = ["WalletRegistry"]
 func.dependencies = [
   "ReimbursementPool",
-  "RandomBeacon",
+  "RandomBeaconChaosnet",
```

Comment:
Trying to confirm my understanding: this is important only for Goerli and local Ethereum instance (client dev, not Hardhat) because for unit tests (Hardhat), the random beacon is replaced with a smock instance here:
https://github.com/keep-network/keep-core/blob/a0817e53bc6ce1c85013ee254d24b88cc78b881a/solidity/ecdsa/test/utils/randomBeacon.ts#L14-L16

In other words, for unit tests, it doesn't matter if we have `RandomBeacon` or `RandomBeaconChaosnet` because it is always a mock.

What about system tests @nkuba @dimpar ?

Reply:
Some unit tests must use the real `RandomBeacon`, but since we decided to use `RandomBeaconChaosnet` only for `goerli` and `development`, the tests will be ok.